### PR TITLE
Add WireMock module to the registry

### PR DESCRIPTION
**Pull request by oleg-nenashev**
FTR https://github.com/wiremock/wiremock-testcontainers-java/issues/53 for tracking on the WireMock side

### CODE REVIEW

File: modules/wiremock.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+            @Test
+            void helloWorld() throws Exception {
+                String url = wiremockServer.getUrl("/hello");
+                HttpResponse response = new TestHttpClient().get(url);
+                assertThat(response.getBody())
+                        .as("Wrong response body")
+                        .contains("Hello, world!");
+            }
+        }
```
Comment (eddumelendez) [inline modules/wiremock.md:34]: Just to be consistent with the other examples

```suggestion
      WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
                .withMapping("hello", WireMockContainerJunit5Test.class, "hello-world.json");
      wiremockServer.start();                
```

File: modules/wiremock.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: WireMock
+categories:
+  - cloud
+  - web
+  - other
```
Comment (eddumelendez) [inline modules/wiremock.md:6]: I think the right would be
```suggestion
  - web
```

File: modules/wiremock.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                String url = wiremockServer.getUrl("/hello");
+                HttpResponse response = new TestHttpClient().get(url);
+                assertThat(response.getBody())
+                        .as("Wrong response body")
+                        .contains("Hello, world!");
+            }
+        }
+      ```
+description: |
+    This module allows provisioning the WireMock server as a standalone container within your unit test,
```
Comment (eddumelendez) [inline modules/wiremock.md:37]:
```suggestion
    This module allows provisioning the WireMock server as a standalone container within your tests,
```

**Review by eddumelendez — CHANGES_REQUESTED** [review]
Thanks for submitting WireMock to the Module Catalog, @oleg-nenashev ! I've left some suggestions

**Comment by oleg-nenashev** [comment]
Hi! Is it required to use just a single category? The fact is that the
module is used for Web API testing, for Cloud API testing and cloud service
integration (WireMock Cloud), and also as a developer tool for many other
cases. So all three categories fit

P.S: A "developer_tools" category would be nice

**Comment by eddumelendez** [comment]
> Is it required to use just a single category?

Not really but can not find another example with multiple categories. I think in this case we are talking about the WireMock itself and its scope and not about how it is provided (Cloud service). Thinking again about `cloud` category, I'm not quite sure TBH because it's not like `LocalStack`.

**Review by kiview — APPROVED** [review]

File: modules/wiremock.md (hunk 2 above)
Comment (oleg-nenashev) [reply modules/wiremock.md:6]: As a suggested compromise, I will remove "other"

```suggestion
  - cloud
  - web
```

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks, @oleg-nenashev ! WireMock is listed in the Module Registry now https://testcontainers.com/modules/wiremock/

**Pull request merged**